test/studies/lcals/LCALSMain.skipif

```diff
@@ -0,0 +1,3 @@
+# This test takes too long if --fast is not used so only run it for
```

Comment:
What about it takes too long? If it's just executing, can we limit it to to only run the short size for correctness testing?